tests/unit/test_tenant.py

```diff
@@ -60,3 +59,8 @@ def test_json(self, create_tenant, create_lease):
 class TestTenantFactory:
     def test_create_tenant(self, create_tenant):
         assert create_tenant()
+
+    def test_create_tenant_with_staff(self, create_tenant, create_join_staff):
+        staff = [create_join_staff().id for _ in range(3)]
+        tenant = create_tenant(staff)
```

Comment:
This is good.
Do you think it would be beneficial to create the staff inside of the create_tenant fixture instead?
Maybe do something like? `create_tenant(staff=3)`. But then there is the downside of not being able to control how the staff is created. Perhaps the create_tenant fixture could be smart. If it's given a list of staff objects then it simply assigns those, if its provided an integer then it creates that many staff members and assigns them. If there are no arguments then it creates a tenant without any assigned staff.

What do you think?